cmd/kubeops/internal/handler/handler_test.go

```diff
@@ -20,11 +20,16 @@ import (
 	"helm.sh/helm/v3/pkg/storage"
 	"helm.sh/helm/v3/pkg/storage/driver"
 	helmTime "helm.sh/helm/v3/pkg/time"
+	"k8s.io/client-go/rest"
 
 	"helm.sh/helm/v3/pkg/release"
 )
 
-const defaultListLimit = 256
+const (
+	defaultListLimit        = 256
+	KUBERNETES_SERVICE_HOST = "KUBERNETES_SERVICE_HOST"
```

Comment:
this would give me a lint warning: `don't use ALL_CAPS in Go names; use CamelCase`
also, for exported consts you need to give it a comment

Reply:
Woops, these were left over from a different approach. Also installed golint in my new env :)